content/en/docs/reference/command-line-tools-reference/feature-gates/DeclarativeValidationTakeover.md

```diff
@@ -0,0 +1,18 @@
+---
+title: DeclarativeValidationMismatchMetric
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: beta
+    defaultValue: false
+    fromVersion: "1.33"
+---
+When enabled, declarative validation errors are returned directly to the caller,
```

Comment:
Change ”declarative validation“ to a link to the docs for this (which you, or a colleague from the SIG, will need to write).

Reply:
I have tentatively pointed this to link to the DeclarativeValidation feature gate file.  I'm happy to write these additional documentation but our current plan was to add such docs in v1.34